INDEX.md

```diff
@@ -0,0 +1,75 @@
+| 2020 | [Improve Activity API usability and OpenTelemetry integration](accepted/2020/diagnostics/activity-improvements.md) | @shirhatti, anksr | @tarekgh |
+| 2020 | [Improve Activity API usability and OpenTelemetry integration (Part 2)](accepted/2020/diagnostics/activity-improvements-2.md) | [Sourabh Shirhatti](https://github.com/shirhatti) | [Tarek Mahmoud Sayed](https://github.com/tarekgh) |
+| 2020 | [Install location search for .NET Core](accepted/2020/install-locations.md) |  | [Vitek Karas](https://github.com/vitek-karas) |
+| 2020 | [JSON extension methods for HttpClient](accepted/2020/json-http-extensions/json-http-extensions.md) | [Immo Landwerth](https://github.com/terrajobst) |  |
+| 2020 | [Linking the .NET Libraries](accepted/2020/linking-libraries.md) |  | [Eric Erhardt](https://github.com/eerhardt) |
+| 2020 | [MSBuild SDK Resolvers and optional workloads in .NET 5](accepted/2020/workloads/workload-resolvers.md) | [Rich Lander](https://github.com/richlander) |  |
+| 2020 | [Marking APIs that are unsupported by Blazor WebAssembly](accepted/2020/blazor-unsupported-apis/blazor-unsupported-apis.md) | [Immo Landwerth](https://github.com/terrajobst) |  |
+| 2020 | [Marking Windows-only APIs](accepted/2020/windows-specific-apis/windows-specific-apis.md) | [Immo Landwerth](https://github.com/terrajobst) |  |
+| 2020 | [OR_GREATER preprocessor symbols for TFMs](accepted/2020/or-greater-defines/or-greater-defines.md) | [Immo Landwerth](https://github.com/terrajobst) |  |
+| 2020 | [Obsoletions in .NET 5](accepted/2020/better-obsoletion/obsoletions-in-net5.md) | [Immo Landwerth](https://github.com/terrajobst) |  |
+| 2020 | [Overview and goals](accepted/2020/serializer/SerializerGoals5.0.md) |  | [Steve Harter](https://github.com/steveharter) |
+| 2020 | [Platform Specific HttpClient Support in dotnet/runtime](accepted/2020/mono-convergence/platform-specific-httpclient.md) |  | [Steve Pfister](https://github.com/steveisok) |
+| 2020 | [Portable PDB](accepted/2020/diagnostics/portable-pdb.md) | [Tomáš Matoušek](https://github.com/tmat) |  |
+| 2020 | [Proposal for global.json](accepted/2020/global-json-updates.md) | [Kathleen Dollard](https://github.com/KathleenDollard) |  |
+| 2020 | [Publishing and Consuming Symbols and Source for Debugging](accepted/2020/diagnostics/debugging-with-symbols-and-sources.md) | [Tomáš Matoušek](https://github.com/tmat) |  |
+| 2020 | [Reproducible Build Validation](accepted/2020/reproducible-builds.md) | [Claire Novotny](https://github.com/clairernovotny) | [Jared Parsons](https://github.com/jaredpar) |
+| 2020 | [Single-file Publish](accepted/2020/single-file/design.md) | [Swaroop Sridhar](https://github.com/swaroop-sridhar) |  |
+| 2020 | [Single-file Publish](accepted/2020/single-file/design_3_0.md) | [Swaroop Sridhar](https://github.com/swaroop-sridhar) |  |
```

Comment:
The names of both designs are "Single-file Publish". Should we rename the 3.0 to something like "Single-file Publish for 3.0"?

Reply:
Sure

Reply:
Fixed in the design file now.